### PR TITLE
TDP: Migrate to import for js-cookie package

### PR DESCRIPTION
**Pull request by anselmbradford**
The `Cookie` export appears to be unused in the JS modules in this PR.

## Removals

- ~~Remove unused `Cookie` export.~~ Turns out these are used in the tests 🤔 We should scrutinize this in the future as this seems like it may be testing `js-cookie`, which isn't our bailiwick.

## Changes

- Migrate to import syntax for js-cookie package.

## How to test this PR

1. PR checks should pass.

### CODE REVIEW

File: cfgov/unprocessed/apps/teachers-digital-platform/js/survey/grade-level-page.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const Cookie = require('js-cookie');
+import Cookies from 'js-cookie';
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/teachers-digital-platform/js/survey/grade-level-page.js:1]: Using `Cookies` vs `Cookie` is just in following the ES Module example on https://www.npmjs.com/package/js-cookie

**Comment by wpears** [comment]
I have performed some jest sorcery so we don't have to export `Cookies` just to import and spy on it

re: the usefulness of these tests, I'd say they're performing more as integration tests than unit tests, but still seem to be a decent thing to check (eg, testing "does this library get called?" when it should makes sense to me)

**Review by wpears — APPROVED** [review]

**Comment by anselmbradford** [comment]
Sorry I still want to review this

File: .eslintrc.js [hunk 2]
```diff
@@ -1,12 +1,6 @@
 module.exports = {
   parser: '@babel/eslint-parser',
   settings: {
-    'import/resolver': {
```
Comment (anselmbradford) [inline .eslintrc.js:4]: This is solely to service the TDP test? How about instead of this, we revert this change and add this to `.eslintrc.js` to make it clearer this is only for TDP:

```
        moduleDirectory: [
          'node_modules',
          // TDP module path is for resolving js-cookie in TDP tests.
          'cfgov/unprocessed/apps/teachers-digital-platform/node_modules',
        ],
```

Comment (anselmbradford) [reply .eslintrc.js:4]: cc @wpears
